config/crd/kustomization.yaml

```diff
@@ -17,13 +17,17 @@ patchesStrategicMerge:
 #- patches/webhook_in_configs.yaml
 #- patches/webhook_in_constraintpodstatuses.yaml
 #- patches/webhook_in_constrainttemplatepodstatuses.yaml
+#- patches/webhook_in_assignmetadata.yaml
```

Comment:
Do we want to add a TODO here to make sure we add in the future?
```yaml
-  bases/mutations.gatekeeper.sh_assignmetadata.yaml
-  bases/mutations.gatekeeper.sh_assign.yaml
```
https://github.com/open-policy-agent/gatekeeper/blob/611327a5539909ab408924149e76dc87b5e4d300/config/crd/kustomization.yaml#L4

Reply:
done